Cant run test suite as root on Ubuntu

When running `make build tests` on Ubuntu as I have always done, I get the following error:

```
./seed-via-docker.sh  -c -t my-ontology1 myont
This script only works with ODK 1.3.2 and later. For ODK 1.3.1 or earlier, use https://raw.githubusercontent.com/INCATools/ontology-development-kit/v1.3.1/seed-via-docker.sh
Traceback (most recent call last):
  File "/tools/odk.py", line 993, in <module>
    cli()
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/tools/odk.py", line 877, in seed
    raise Exception('max one repo; current={}'.format(repo))
Exception: max one repo; current=('Matentzoglu', 'myont')
make[2]: *** [Makefile:22: test_no_yaml_dependencies_none] Error 1
make[2]: Leaving directory '/root/ws/ontology-development-kit'
make[1]: *** [Makefile:110: test-flavor] Error 2
make[1]: Leaving directory '/root/ws/ontology-development-kit'
```

This has always worked, and I don't have any indications of what could be wrong..

... Other than perhaps the fact that my server does not have any user management and only a root user..

--- a/scripts/entrypoint.sh
+++ b/scripts/entrypoint.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 if [ x$ODK_USER_ID = x0 ]; then
-    exec $@
+    exec "$@"
 fi
 
 REAL_USER_ID=${ODK_USER_ID:-1000}

